onetl/connection/db_connection/kafka/connection.py

```diff
@@ -352,11 +354,52 @@ def get_packages(
     def instance_url(self):
         return "kafka://" + self.cluster
 
+    @root_validator(pre=True)
+    def _get_addresses_by_cluster(cls, values):
+        cluster = values.get("cluster")
+        addresses = values.get("addresses")
+        if not addresses:
+            cluster_addresses = cls.Slots.get_cluster_addresses(cluster) or []
+            if cluster_addresses:
+                log.debug("|%s| Set cluster %r addresses: %r", cls.__name__, cluster, cluster_addresses)
+                values["addresses"] = cluster_addresses
+            else:
+                raise ValueError("Passed empty parameter 'addresses'")
+        return values
+
+    @validator("cluster")
+    def _validate_cluster_name(cls, cluster):
+        log.debug("|%s| Normalizing cluster %r name ...", cls.__name__, cluster)
+        validated_cluster = cls.Slots.normalize_cluster_name(cluster) or cluster
+        if validated_cluster != cluster:
+            log.debug("|%s|   Got %r", cls.__name__, validated_cluster)
+
+        log.debug("|%s| Checking if cluster %r is a known cluster ...", cls.__name__, validated_cluster)
+        known_clusters = cls.Slots.get_known_clusters()
+        if known_clusters and validated_cluster not in known_clusters:
+            raise ValueError(
+                f"Cluster {validated_cluster!r} is not in the known clusters list: {sorted(known_clusters)!r}",
+            )
+
+        return validated_cluster
+
     @validator("addresses")
-    def _validate_addresses(cls, value):
-        if not value:
-            raise ValueError("Passed empty parameter 'addresses'")
-        return value
+    def _validate_addresses(cls, value, values):
+        cluster = values.get("cluster")
+
+        log.debug("|%s| Normalizing addresses %r names ...", cls.__name__, value)
+
+        validated_addresses = [cls.Slots.normalize_address(address, cluster) or address for address in value]
+        if validated_addresses != value:
+            log.debug("|%s| Got %r", cls.__name__, validated_addresses)
+
+        cluster_addresses = set(cls.Slots.get_cluster_addresses(cluster) or [])
+        unknown_addresses = {address for address in validated_addresses if address not in cluster_addresses}
+        if cluster_addresses and unknown_addresses:
+            log.error("|%s| Addresses for cluster %r: %r", cls.__name__, cluster, cluster_addresses)
+            raise ValueError(f"Addresses {', '.join(unknown_addresses)} are not in the cluster")
```

Comment:
```suggestion
            raise ValueError(f"Cluster {cluster!r} does not contain addresses {unknown_addresses!r}")
```
Нет особого смысла логировать ошибку, и сразу же выбрасывать ее же в виде исключения